app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -70,6 +76,20 @@ enum class NavigationItem(
         content = { UnderConstructionScreen("Create Team Screen") }
     ),
 
+    JoinTeam(
+        primaryRoute = JOIN_TEAM,
+        canonicalRoute = JOIN_TEAM,
+        arguments = listOf(navArgument(EXTRA_TEAM_CODE) {
+            type = NavType.StringType
+        }),
+        deepLinks = listOf(navDeepLink {
+            uriPattern = TEAM_INVITATION_URI
+        }),
+        content = { contentParams ->
+            val teamCode = contentParams.navBackStackEntry.arguments?.getString(EXTRA_TEAM_CODE)
+            UnderConstructionScreen("Join Team Screen via InvitationLink [Your team code is $teamCode")
```

Comment:
good candidate to extract it to string resource

Reply:
Agree, but this is only for the under-constructer text! So that will be deleted when the team screen get merged.